app/src/main/java/edu/unikom/herbamedjabar/view/LoginFragment.kt

```
@@ -32,15 +29,17 @@ import kotlinx.coroutines.launch
 class LoginFragment : Fragment() {
 
     private var _binding: FragmentLoginBinding? = null
-    private val binding get() = _binding!!
+    private val binding
+        get() = _binding!!
 
     private val viewModel: AuthViewModel by viewModels()
 
     private lateinit var credentialManager: CredentialManager
```

detekt (1.23.8)

[warning] Usages of lateinit should be avoided.

(detekt.potential-bugs.LateinitUsage)